README.md

```diff
@@ -62,20 +62,21 @@ The filter expression can be any valid python expression that evaluates to `bool
 ### Available fields
 The following VCF fields can be accessed in the filter expression:
 
-|Name|Type|Interpretation|Example expression|
-|---|---|---|---|
-|`INFO`|`Dict[str, Any¹]`| `INFO field -> Value`  | `INFO["DP"] > 0`|
-|`ANN`| `Dict[str, Any²]`| `ANN field -> Value` | `ANN["Gene_Name"] == "CDH2"`|
-|`CHROM`| `str` | Chromosome Name  |  `CHROM == "chr2"` |
-|`POS`| `int` | Chromosomal position  | `24 < POS < 42`|
-|`ID`| `str`  | Variant ID |  `ID == "rs11725853"` |
-|`REF`| `str` |  Reference allele  | `REF == "A"` |
-|`ALT`| `str` |  Alternative allele³  | `ALT == "C"`|
-|`QUAL`| `float`  | Quality |  `QUAL >= 60` |
-|`FILTER`| `List[str]` | Filter tags | `"PASS" in FILTER` |
-|`FORMAT`|`Dict[str, Dict[str, Any¹]]`| `Format -> (Sample -> Value)` | `FORMAT["DP"][SAMPLES[0]] > 0` |
-|`SAMPLES`|`List[str]`| `[Sample]`  |  `"Tumor" in SAMPLES` |
-|`INDEX`|`int`| Index of variant in the file  |  `INDEX < 10` |
+| Name      | Type                         | Interpretation                     | Example expression             |
+| --------- | ---------------------------- | ---------------------------------- | ------------------------------ |
+| `INFO`    | `Dict[str, Any¹]`            | `INFO field -> Value`              | `INFO["DP"] > 0`               |
+| `ANN`     | `Dict[str, Any²]`            | `ANN field -> Value`               | `ANN["Gene_Name"] == "CDH2"`   |
+| `CHROM`   | `str`                        | Chromosome Name                    | `CHROM == "chr2"`              |
+| `POS`     | `int`                        | Chromosomal position (1-based)     | `24 < POS < 42`                |
+| `END`     | `int`                        | Chromosomal end position (1-based) | `24 < END < 42`                |
```

Comment:
I'm not sold on exposing `END` in addition to `INFO["END"]` yet. Every entry in this table is related to an actual VCF column, with the exceptions of `ANN` (which is core to vembrane) and `INDEX` which just keeps track of record indices. `END` would be the first truly derived property, which is either `POS + len(REF)` or `INFO["END"]`. But at least it's a reserved INFO key in the VCF spec v4.3.

Reply:
Indeed. But it makes sense to me because it would feel not natural to access INFO["END"] for a small variant where there is no END tag in the record at all.

Reply:
Yes, that is true, I'm okay with having `END` exposed then.
What happens for breakends across contigs?

Reply:
For breakends, having END in INFO makes little sense. I've checked it though: pysam would use whatever value is defined there. If you don't specify END for a breakend, it will be the breakend position itself, like with an SNV.

Reply:
We could alternatively set END to NA in case of a breakend.

Reply:
I think that should be more correct. What do you think?

Reply:
Semantically, I think NA makes more sense, because it's difficult or impossible to know what the "end of a variant" in the context of breakends actually is.

Reply:
Can you update the README accordingly?

Reply:
done